lmvachon/bad.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+
+<head>
+	<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+	<title>Lauren's Bad Code Example</title>
+
+	<link rel="stylesheet" href="/css/style.css" type="text/css" media="all" />
+</head>
+
+<body>
+
+	<header>
+		<nav>
+			<a href="#">Home
+			<a href="#">About
+			<a href="#">Contact
+		</nav>
+	</header>
+	<section>
+		<article>
+			<header>
```

Comment:
Do we need this `header` tag? I think it depends on whether we anticipate additional introductory elements in this section or not. If it's just some header elements and some paragraphs, wrapping these in a `header` tag may be overkill. See: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header